frameworks/hdfs/alerts/HDFS_alert_rules.yml

```diff
@@ -0,0 +1,105 @@
+groups:
+  - name: hdfs_info
+    rules:
+      # Alert for any instance that has a median request latency > 1s.
+    - alert: APIHighRequestLatency
+      expr: api_http_request_latencies_second{quantile="0.5"} > 1
+      for: 10m
+      labels:
+        severity: info
+        service: hdfs
+      annotations:
+        summary: "High request latency on {{ $labels.instance }}"
+        description: "{{ $labels.instance }} has a median request latency above 1s (current value: {{ $value }}s)"
+
+  - name: hdfs_warning
+    rules:
+    - alert: CapacityRemainingGB
+      expr: sum by (service_name,task_name) ({__name__=~".*_FSNamesystem_CapacityRemainingGB"}) < 10
+      for: 10m
+      labels:
+       severity: warning
+       service: hdfs
+      annotations:
+       summary: "Low capacity remaining in GB on {{ $labels.instance }} for {{ $labels.service_name }} / {{ $labels.task_name }}"
+       description: "{{ $labels.instance }} / {{ $labels.service_name }} / {{ $labels.task_name }} has capacity remaining {{ $value }}GB"
+
+    - alert: MissingBlocks
+      expr: sum by(service_name,task_name) (rate({__name__=~".*_dfs_FSNamesystem_MissingBlocks"}[10m])) > 0
+      labels:
+        severity: warning
+        service: hdfs
+      annotations:
+       summary: "Missing blocks on {{ $labels.instance }} for {{ $labels.service_name }} / {{ $labels.task_name }}"
+       description: "{{ $labels.instance }} / {{ $labels.service_name }} / {{ $labels.task_name }} missed blocks equals to {{ $value }}"
+
+    - alert: UnderReplicatedBlocks
+      expr: sum by(service_name,task_name) (rate({__name__=~".*_dfs_FSNamesystem_UnderReplicatedBlocks"}[10m])) > 0
+      labels:
+        severity: warning
+        service: hdfs
+      annotations:
+       summary: "Under replicated blocks on {{ $labels.instance }} for {{ $labels.service_name }} / {{ $labels.task_name }}"
+       description: "{{ $labels.instance }} / {{ $labels.service_name }} / {{ $labels.task_name }} has under replicated blocks equals to {{ $value }}"
```

Comment:
These descriptions and summaries are merely repeats of each other. There's no point to say the same thing twice. As I commented previously, it would be more helpful to provide a clue or some helpful instruction to the person receiving the alert.